website/content/docs/auth/jwt.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: JWT - Auth Method
+description: >-
+  The JWT auth method allows authentication using signed JWTs
+---
+
+# JWT auth method
+
+@include 'x509-sha1-deprecation.mdx'
+
+The `jwt` auth method can be used to authenticate with Vault using JSON Web Tokens
+([JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)). This form of authentication
+is commonly used in Workload Identity Federation (WIF) workflows for machine-to-machine
+authentication.
+
+The auth method is configured to trust a JWT issuer. For example, this could be a
+cloud service provider that makes workload identity available in a virtual machine
+or CI provider that provides identity to jobs. It must be [configured](#jwt-verification)
+with public keys from the JWT issuer by either OIDC Discovery, a JSON Web Key Set (JWKS) URL,
+or locally-provided keys. A signed JWT from the configured issuer is required as the form
+of authentication. The auth method will cryptographically verify the signature using the
+issuer's public key. The verified JWT claims can be used to provide [further validation](#bound-claims)
+to gate authentication to Vault.
```

Comment:
```suggestion
To use JWT, you must configure Vault to trust a JWT issuer. For example, a
cloud service provider that makes workload identity available in a virtual machine
or CI provider that provides identities to jobs.

Vault will require a signed JWT from the configured issuer as the form of
authentication and the authN method will cryptographically verify the client
signature using the issuer's public key. You can use verified JWT claims to
gate authentication to Vault by requiring [additional validation](#bound-claims).

You must [configure](#jwt-verification) the authN method with public keys from
the JWT issuer with OIDC Discovery, a JSON Web Key Set (JWKS) URL, or
locally-provided keys. 
```

Style correction: write in active voice, make content scannable, make it clear who/what takes action